Sources/Shared/Notifications/NotificationCommands/NotificationsCommandManager.swift

```diff
@@ -104,8 +104,8 @@ private struct HandlerReloadWidgets: NotificationCommandHandler {
         Current.Log.verbose("Reloading widgets triggered by notification command")
         return Promise<Void> { seal in
             DispatchQueue.main.async {
-                WidgetCenter.shared.reloadTimelines(ofKind: "io.robbie.HomeAssistant.widget-gauge")
-                WidgetCenter.shared.reloadTimelines(ofKind: "io.robbie.HomeAssistant.widget-details")
+                WidgetCenter.shared.reloadTimelines(ofKind: "WidgetGauge")
```

Comment:
Can you make a variable/enum or just something to keep changes aligned between Widget Gauge, WidgetDetails and NotificationsCommandManager?

Just so that if we change the string in one of this places the other one will also get updated automatically.